src/com/company/CricketMatch.java

```diff
@@ -0,0 +1,221 @@
+package com.company;
+import java.util.List;
+
+public class CricketMatch {
+    Player strikerBatsman;
+    Player nonStrikerBatsman;
+    Player strikeBowler;
+    Team battingTeam;
+    Team bowlingTeam;
+    Team firstBattingTeam;
+    Team secondBattingTeam;
+    private void swapPlayer()
+    {
+        Player temp = strikerBatsman;
+        strikerBatsman = nonStrikerBatsman;
+        nonStrikerBatsman = temp;
+    }
+
+    private void swapTeam()
+    {
+        Team temp = battingTeam;
+        battingTeam = bowlingTeam;
+        bowlingTeam = temp;
+    }
+
+    private void printWicketCommentary()
+    {
+        System.out.println("Wicket gone!");
+        System.out.println("--------------------------------------------------------------------------");
+        System.out.println(strikerBatsman.getName() + "\t" + strikerBatsman.getRole() + "\nRun - " + strikerBatsman.getRunScore() + "(" + strikerBatsman.getNoOfBallsPlayed() +")");
+        System.out.println("Team Score - " + battingTeam.getRun() + " \t Wickets - " + battingTeam.getWickets());
+        System.out.println("--------------------------------------------------------------------------");
+    }
+
+    private void printCommentary(int run)
+    {
+        System.out.println(run + "\t Total Run - " + battingTeam.getRun());
+    }
+
+    private void updateScoreBoard(int run)
+    {
+        strikerBatsman.addRunScore(run);
+        strikerBatsman.addNoOfBallsPlayed(1);
+        battingTeam.addRun(run);
+        strikeBowler.addGivenRun(run);
+        if(run % 2 == 1)swapPlayer();
+    }
+
+    private void wicketGone()
+    {
+        strikerBatsman.setBattingStatus(PlayerBattingStatus.status3.getValue());
+        strikerBatsman.addNoOfBallsPlayed(1);
+        battingTeam.addWickets(1);
+        strikeBowler.addNumberOfWicketsTaken(1);
+        printWicketCommentary();
+
+        if(battingTeam.getWickets() >= 10){
+            return;
+        }
+
+        if(strikerBatsman.getId() > nonStrikerBatsman.getId())
+        {
+            strikerBatsman = battingTeam.getPlayerList().get(strikerBatsman.getId() + 1);
+        }
+        else
+        {
+            strikerBatsman = battingTeam.getPlayerList().get(nonStrikerBatsman.getId() + 1);
+        }
+
+        strikerBatsman.setBattingStatus(PlayerBattingStatus.status2.getValue());
+    }
+
+    private String randomGenerator()
+    {
+        String[] possibleOutput = {"0","1","2","3","4","5","6","Wide","NoBall","Wicket"};
+        return possibleOutput[(int)(Math.random() * 10)];
+    }
+
+    public void tossOfMatch(List<Team> teams)
+    {
+        System.out.println("\n-----------------------------------------------------------------------------");
+        int tossWinTeam = (int)(Math.random()*2);
+        int choiceWinTeam = (int)(Math.random()*2);
```

Comment:
good that you randomized even the choice selection. But reading the variable name choiceWinTeam does not tell me what it means. I have to read your code to understand that 0 means batting and 1 means bowling. 
this is actually a good use of enum. create an enum with two values of Bat and Bowl, and randomize from that enum.